usagov/usagov-2021
# USAGOV-1216 Allow CMS users to check status of static site generation

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1216

## Description


Added a script to create/update a text file which will contain a basic Tome status/timestamp message for CMS users.

Add calls to the above script in tome-run and tome-sync scripts.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



After this ticket is deployed, and at least one tome run has started

- navigate to the /static-site-status.txt on the CMS URL
- you should see a page containing tome run status.

This should not be available from the static site URL.   So /static-site-status.txt on the static site URL should return a 404.

### Requires New Config
- [c] No

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review



## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
